images/win/scripts/Installers/Run-Antivirus.ps1

```diff
@@ -0,0 +1,38 @@
+################################################################################
+##  File:  Run-Antivirus.ps1
+##  Desc:  Run a full antivirus scan.
+##         Run right after cleanup before we sysprep
+################################################################################
+
+try{
+    Update-MpSignature
+}
+catch {
+    Write-Host "Some error was found"
+    Write-Host $_
+}
+
+if($env:$run_scan_antivirus -eq $true) {
```

Comment:
Should be $env:run_scan_antivirus

Reply:
changed $env:$run_scan_antivirus to $env:run_scan_antivirus